src/CMakeLists.txt

```diff
@@ -55,12 +55,12 @@ install(
   TARGETS ${LIB_TARGETS}
   EXPORT ${PROJECT_NAME}Exports
   RUNTIME DESTINATION bin
```

Comment:
Should something similar be done to "bin"?

Reply:
Yes.  It wasn't critical for my use case, but we should be consistent.  Added.